app_dart/lib/src/request_handlers/scheduler/batch_backfiller.dart

```diff
@@ -111,6 +115,26 @@ class BatchBackfiller extends RequestHandler {
     }
   }
 
+  // Remove targets that specify a property of 'backfill': 'false'. Otherwise we
+  // will assume that we include those targets when backfilling.
+  List<Target> removeNonBackFillTargets(List<Target> targets) {
+    final List<Target> filteredTargets = <Target>[];
+    for (Target target in targets) {
+      final Map<String, String> properties = target.value.properties;
```

Comment:
Instead we should use `target.getProperties()`.

Reply:
Sounds good.